docs/sql_reference/src/asciidoc/_chapters/sql_functions_and_expressions.adoc

```diff
@@ -5367,6 +5367,664 @@ insert into tlob130txt1 values(1,filetolob('lob_input_a1.txt'));
 update tlob1 set c2=filetolob('hdfs:///lobs/lob_update.txt', append) where c1 = 3;
 ```
 
+[[floor_function]]
+== FLOOR Function
+
+The FLOOR function rounds down `_numeric-expression_` to the next largest integer less than or equal to `_numeric-expression_`.  
```

Comment:
Minor wordsmith: "The FLOOR function rounds down a `_numeric-expression_` to the largest integer less than or equal to `_numeric-expression_`."  (Add "a", delete "next")

Reply:
@DaveBirdsall Thank you, Dave. I've incorporated your comment.